src/get-started/focus-states/index.md.njk

```diff
@@ -0,0 +1,83 @@
+
+The yellow has a high contrast with dark backgrounds and the thick black border has a high contrast against light backgrounds.
+
+## Link focus state style
+
+When links are focused, they have a yellow background with a black bottom border. This helps the focused link stand out from the rest of the content on the page.
+
+![A focused link against different GOV.UK background colours](link-focus.png)
+
+Other components and elements that look like links use the link focus state style. For example, the controls on the [accordion](/components/accordion/) and [details](/components/details/) components.
+
+![A focused details component. In the examples, the expandable text reads "Help with nationality" and beneath is an explanation of why the user is being asked for this information.](details-focus.png)
+
+## Form input focus state style
+
+When form inputs are focused, they have a yellow outline and a thick black border. If the element already has a border, the border gets thicker.
+
+![A text input labelled "What is your name?". The example shows the text input both unfocused and focused.](text-input-focus.png)
+
+[Radios](/components/radios/) and [checkboxes](/components/checkboxes/) use the same style.
+
+![Yes and no radio options to answer the question "Have you changed your name?". In this example, the "No" option is focused.](radios-focus.png)
+
+## Making focus states accessible for extended and modified components
+
+If you've [extended or modified components in the GOV.UK Design System](/get-started/extending-and-modifying-components/), you can use GOV.UK styles to make the focus states of these components accessible.
+
+How you make focus states accessible depends on if the component is:
+
+- focusable text without a background colour or border
+- another focusable element with a background colour or border
+
+### Make focusable text accessible
+
+If you use Sass, you should include the `govuk-focused-text` mixin in your component's `:focus` selector if that component is focusable text. For example, the component is a link in body text, or the details component:
+
+```
+.app-component:focus {
+@include govuk-focused-text;
+}
+```
+### Make other focusable elements accessible
+
+If you use Sass, you can use 3 GOV.UK Frontend variables if your component has a background colour or border. For example a text input or checkbox.
+
+The 3 Sass variables are:
+
+- `govuk-focus-colour` - yellow background
+- `govuk-focus-text-colour` - black text 
+- `govuk-focus-width` - for consistent width
+
+Use these variables in your components instead of numeric values for the background, text and widths.
+
+To test the focus states in your components are still accessible, you can:
```

Comment:
This seems to come out of nowhere – we haven't talked about supporting users changing to use different colours, nor what it is they should be looking for if they do test this?

Reply:
I think the original draft had this but we may have simplified it too much.

Reply:
Do we need to remove it or add more detail to resolve this? 

Ideally these kinds of questions should be picked up before 2i is done, so let's have a chat next week about how we can make sure that's happening.

Reply:
I suggest we just get rid of this sentence and the following bullet points to get this out the door, and revisit this later.

I'm a little concerned about suggesting that service teams _have_ to make sure that things work when colours are overridden as it's not written down anywhere else (e.g. the service manual), so if we do add something about this we might need to make it softer.

Reply:
Removed this section for now.